src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/MIReportingService.java

```diff
@@ -67,12 +67,9 @@ public void logicalReplicationCheck() {
         Objects.requireNonNull(taskResourceRepository, "Primary Task DB repo is null.");
         Objects.requireNonNull(taskResourceRepository, "Replica Task DB repo is null.");
 
-        if (!taskResourceRepository.showWalLevel().equals(WAL_LEVEL)
-            || !reportableTaskRepository.showWalLevel().equals(WAL_LEVEL)) {
-
-            log.error("WAL LEVEL for primary DB; {}, replicaDB: {}.  These must be set to logical",
-                taskResourceRepository.showWalLevel(),
-                reportableTaskRepository.showWalLevel());
+        if (!taskResourceRepository.showWalLevel().equals(WAL_LEVEL)) {
```

Comment:
May be Rename of Constant WAL_LEVEL to something like WAL_LEVEL_LOGICAL may make it more readable.

Reply:
👍